patterns/java/src/refactoring_guru/patterns/cash_prototype/example/Application.java

```diff
@@ -0,0 +1,14 @@
+package refactoring_guru.patterns.cash_prototype.example;
+
+import refactoring_guru.patterns.prototype.example.Rectangle;
+
+public class Application {
+    public static void main(String[] args) {
+        PrototypeCash cash = new PrototypeCash();
+        cash.loadCash();
+        Rectangle rectangle = (Rectangle)cash.getByType("Rectangle");
+        cash = null;
```

Comment:
Are these two lines suppose to show that created rectangle is independent from a cache object? If so, that is not really obvious. How about requesting two rectangles and comparing them?